### PR TITLE
complete higher lower game

### PR DESCRIPTION
**Pull request by mercurrie**
created art.py: contains various ASCII artwork
created game_data.py: contains data to be used in main.py
created main.py: contains vital functions to complete program
included more information to README.md

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+# compare() calculates which choice has more followers and checks if user's answer is that choice. Returns bool
+def compare(choice_a: dict, choice_b: dict) -> bool:
+    user_answer = input('Who has more instagram followers? Type "A" or "B": ').lower()
+    
+    if choice_a['follower_count'] > choice_b['follower_count']:
+        solution = 'a'
+    elif choice_b['follower_count'] > choice_a['follower_count']:
+        solution = 'b'
+    else:
```
Comment (michaelhousman) [inline main.py]: Not sure what this `else` is for.  Does this handle the scenario where the follower count is equal?

File: main.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import random
+import art
+import os
+from game_data import data
+
+# compare() calculates which choice has more followers and checks if user's answer is that choice. Returns bool
+def compare(choice_a: dict, choice_b: dict) -> bool:
+    user_answer = input('Who has more instagram followers? Type "A" or "B": ').lower()
+    
+    if choice_a['follower_count'] > choice_b['follower_count']:
```
Comment (michaelhousman) [inline main.py]: See if you can do this in one line:

solution = 'a' if <condition> else 'b'

File: main.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        solution = 'a'
+    elif choice_b['follower_count'] > choice_a['follower_count']:
+        solution = 'b'
+    else:
+        solution = user_answer
+        
+    if user_answer == solution:
```
Comment (michaelhousman) [inline main.py]: One line to replace these 4:

return user_answer == solution

Try it.

File: main.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    end = compare(choice_a = subject1, choice_b = subject2)
+    score = 0
+    
+    while end:
+        os.system('cls')
+        score += 1
+        print(f"You're right! Current score: {score}.")
+        subject1 = subject2
+        subject2 = random.choice(data)
+        print_UI(subject1 = subject1, subject2 = subject2)
```
Comment (michaelhousman) [inline main.py]: Don't mix snake_case and capital letters

File: game_data.py [hunk 5]
```diff
@@ -0,0 +1,302 @@
+data = [
```
Comment (michaelhousman) [inline game_data.py]: This is good.   For practice, try converting this to JSON and importing it in the Python script.

**Review by michaelhousman — CHANGES_REQUESTED** [review]
This is good.  Left you a few things to work on.

File: main.py (hunk 1 above)
Comment (mercurrie) [reply main.py]: That is correct. The else is to handle the case the follower counts are equal

File: main.py (hunk 2 above)
Comment (mercurrie) [reply main.py]: recommended changes made

File: main.py (hunk 3 above)
Comment (mercurrie) [reply main.py]: Ah I understand

File: main.py (hunk 4 above)
Comment (mercurrie) [reply main.py]: Mental note taken. will only code lowercase snake_case

File: game_data.py (hunk 5 above)
Comment (mercurrie) [reply game_data.py]: I converted 'game_data.py' into 'game_data.json', imported the file in 'main.py', opened the file, loaded it to a variable, and closed the file at the end of the program

Comment (michaelhousman) [reply game_data.py]: Excellent

File: main.py (hunk 4 above)
Comment (michaelhousman) [reply main.py]: Cool

File: main.py (hunk 2 above)
Comment (michaelhousman) [reply main.py]: Great

**Review by michaelhousman — APPROVED** [review]
This one looks good.  You can go ahead and merge :)

**Pull request merged**